webapp/src/app.tsx

```diff
@@ -320,18 +320,8 @@ export class ProjectView
         data.invalidate('pkg-git-pr');
         data.invalidate('pkg-git-pages')
 
-        // there is a race with another tab that has a lock on the device
```

Comment:
This hack was in place in the case of using webusb in MakeCode.com/multi . Probably worth testing this change in that page too

Reply:
Is there a way to test that locally?

Reply:
@pelikhan what is the expected behavior here? As far as I can tell it works the same as the current site, only one of the frames is able to claim the webusb and you can't switch between them.

Reply:
i can repro what was the issue. i think you overall change might just make this hack irrevelant anymore. 
Do you handle the case where 2 different microbits are connect to different tabs? This scenari works currently in multi.

Reply:
I don't think that would work, right now its a lock on webUSB as a whole

Reply:
I guess it's not a core scenario but it's a bit unfortunate.

Reply:
lemme take a look at how hard it would be to support that